fastestimator/estimator.py

```diff
@@ -65,6 +66,7 @@ def __init__(self,
         self.trace_inputs = dict()
 
     def fit(self):
+        draw()
```

Comment:
this call is in the logger now since we want user to pass None to disable all console output right?

Reply:
not this one, because this is our signature :)